great_expectations/datasource/data_connector/runtime_data_connector.py

```diff
@@ -47,77 +41,52 @@ def __init__(
         )
 
         self._runtime_keys = runtime_keys
+        self._refresh_data_references_cache()
 
     def _refresh_data_references_cache(self):
-        """"""
-        # Map data_references to batch_definitions
-        data_reference: str = self._get_data_reference_list()[0]
-        mapped_batch_definition_list: List[
-            BatchDefinition
-        ] = self._map_data_reference_to_batch_definition_list(
-            data_reference=data_reference, data_asset_name=None
-        )
-        self._data_references_cache = {data_reference: mapped_batch_definition_list}
+        self._data_references_cache = {}
 
     def _get_data_reference_list(
         self, data_asset_name: Optional[str] = None
     ) -> List[str]:
-        """List objects in the underlying data store to create a list of data_references.
-
-        This method is used to refresh the cache.
-        """
```

Comment:
I wonder if a docstring here would be helpful.

Reply:
added